### ISSUE TITLE
Universal scrobbler

### ISSUE THREAD

**Issue by nicoleahmed** [hagezi/dns-blocklists#1933]
### Which AdBlocker/DNS cloud service do you use?

NextDNS

### NextDNS users only

- [X] I have disabled the block page in NextDNS and confirm that the problem still occurs!

### Which blocklist(s) do you use?

Multi LIGHT

### Which domain(s) should be unblocked?

Universalscrobbler.com
Discogs.com

### Why should the domain(s) be unblocked?

Unsure why these are blocked. Tested by removing hagezi light (issue resolved), then readding (issue reappeared) then whitelisting both universalscrobbler.com and discogs.com

Universal Scrobbler is a last.fm music track scrobbling site
Discogs is a site for managing records of music releases

Thanks

**Comment by hagezi** [comment]
@nicoleahmed 

I can't find either domains on any of my list versions. They are not blocked by my lists.

Have a look at your NextDNS log.

**Comment by nicoleahmed** [comment]
Thank you

I'm just starting to use next dns. It was definitely the addition of the light list that changed the behaviour. Perhaps when I've whitelisted those domains in next dns (which has fixed functioning) it has over ruled associated requests which were being blocked by the list? Thank you for looking. I've whitelisted and it's working for me now. I looked through the logs but there were several blocks (not the domains I mentioned as you said) so I'm not sure what would need whitelisting to fix for others. Thanks anywau
